includes/services/PageManager.php

```diff
@@ -499,11 +499,18 @@ private function checkEntriesACL(array $pages, ?string $tag = null, ?string $use
         }
         $pages = array_map(function ($page) use ($guard, $allEntriesTags, $userNameForCheckingACL) {
             return (isset($page['tag']) &&
-                    in_array($page['tag'], $allEntriesTags)
+                in_array($page['tag'], $allEntriesTags)
             ) ? $guard->checkAcls($page, $page['tag'], $userNameForCheckingACL)
-                    : $page;
+                : $page;
         }, $pages);
 
         return $pages;
     }
+
+    private function duplicate($sourceTag, $destinationTag): bool
```

Comment:
?? un code de test? à supprimer non?

Reply:
non je voulais creer une methode dans le PageManager, mais work in progress